node/outgoing_response.js

```diff
@@ -111,13 +117,12 @@ TChannelOutgoingResponse.prototype.sendCallResponseFrame = function sendCallResp
     var self = this;
     switch (self.state) {
         case States.Initial:
+            self.start = self.timers.now();
             self.sendFrame.callResponse(args, isLast);
-            if (isLast) {
-                if (self.span) {
-                    self.span.annotate('ss');
-                }
-                self.state = States.Done;
+            if (self.span) {
+                self.span.annotate('ss');
```

Comment:
cc @rf since this is a semantic change, but one for the better and one that doesn't make much difference right now anyhow